src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/ConfigurationExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static IConfigurationBuilder Add<TSource>(this IConfigurationBuilder buil
         /// <summary>
         /// Shorthand for GetSection("ConnectionStrings")[name].
         /// </summary>
-        /// <param name="configuration">The configuration.</param>
+        /// <param name="configuration">The <see cref="IConfiguration"/> to enumerate.</param>
```

Comment:
```suggestion
        /// <param name="configuration">The configuration to enumerate.</param>
```

There's no need to include the explicit type. The type is added automatically in the docs and is visible in IntelliSense. The documentation is here if you're interested: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors.